test/functional/http/post/transactions.js

```diff
@@ -43,4 +56,24 @@ describe('POST /api/transactions (general)', () => {
 				);
 			});
 	});
+
+	describe('transaction processing', () => {
+		sendTransactionPromise(transaction).then(res => {
+			expect(res.body.data.message).to.be.equal('Transaction(s) accepted');
+		});
+		return phases.confirmation([transaction], []);
+	});
+
+	describe('validation', () => {
```

Comment:
consistency - `verification` or `validation`, globally

Reply:
Thanks, @4miners. Actually, @nazarhussain suggest to remove all the individual tests and keep only verification test here`test/functional/http/post/transactions.js`, I will remove the repetitive tests.